kolibri/core/assets/src/views/CoreBase/ScrollingHeader.vue

```diff
@@ -39,6 +39,11 @@
         type: Boolean,
         default: false,
       },
+      // This can be used to override automatic hiding/showing updates
```

Comment:
I'd like to move this into `ScrollingHeader` so that it's not necessary to set it externally, but haven't gotten it quite figured out yet.